binary_search_tree/binary_search_tree.py

```diff
@@ -41,17 +42,16 @@ def contains(self, target, leaf=None):
                 if target == leaf:
                     print("we have a winner: ", leaf)
                     return True
+
                 else:
                     return False
         else:
             return False
 
```

Comment:
Great work so far , just remember to keep your code tidy especially since python will shoot out errors due to indentation.